kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -107,25 +107,26 @@ namespace: default
 # Java Option for WebLogic Server
 javaOptions: -Dweblogic.StdoutDebugEnabled=false
 
-# Name of the persistent volume claim 
+# Name of the persistent volume claim
 # If not specified, the value is derived from the domainUID as <domainUID>-weblogic-sample-pvc
 # This parameter is required if 'logHomeOnPV' is true.
 # Otherwise, it is ignored.
 persistentVolumeClaimName: domain1-weblogic-sample-pvc
 
-# Mount path of the domain persistent volume. 
+# Mount path of the domain persistent volume.
 # This parameter is required if 'logHomeOnPV' is true.
 # Otherwise, it is ignored.
 domainPVMountPath: /shared
 
 # Base WebLogic binary image used to build the WebLogic domain image
 # The operator requires WebLogic Server 12.2.1.3.0 with patch 29135930 applied.
+# The WebLogic Docker image, `store/oracle/weblogic:12.2.1.3`, meets this requirement.
```

Comment:
@rosemarymarano, Given that we updated the existing tag it's likely that some customers already have store/oracle/weblogic:12.2.1.3 pulled locally, but need to `docker pull` again to get the latest.  Perhaps a message that says, "The WebLogic Docker image, `store/oracle/weblogic:12.2.1.3`, was updated on January, 17th, 2019 and now meets this requirement."

Reply:
Agreed.